### PR TITLE
fix: enhancement cluster code analysis

### LINKED ISSUE #615: Enhancement: gitee quality analyses -- cluster
**What would you like to be added**:

**Why is this needed**:

### PR DESCRIPTION
**Pull request by georgehao**
**What this PR does**:
Enhancement cluster gitee code analysis

**Which issue(s) this PR fixes**:

Fixes #615

### CODE REVIEW

File: cluster/cluster_impl/failback_cluster_invoker.go [hunk 1]
```diff
@@ -91,26 +91,25 @@ func (invoker *failbackClusterInvoker) process(ctx context.Context) {
...
-
-				retryInvoker := invoker.doSelect(retryTask.loadbalance, retryTask.invocation, retryTask.invokers, invoked)
-				var result protocol.Result
-				result = retryInvoker.Invoke(ctx, retryTask.invocation)
-				if result.Error() != nil {
-					retryTask.lastInvoker = retryInvoker
-					invoker.checkRetry(retryTask, result.Error())
-				}
-			}(retryTask)
+func (invoker *failbackClusterInvoker) tryTimerTaskProc(ctx context.Context, retryTask *retryTimerTask) {
```
Comment (AlexStocks) [inline cluster/cluster_impl/failback_cluster_invoker.go:103]: pls move this func up the process.

File: cluster/router/condition/app_router_test.go [hunk 2]
```diff
@@ -18,7 +18,8 @@
 package condition
 
 import (
-	"strconv"
+	"fmt"
+	"github.com/apache/dubbo-go/common/constant"
```
Comment (zouyx) [inline cluster/router/condition/app_router_test.go:22]: Split it

File: cluster/router/condition/factory_test.go [hunk 3]
```diff
@@ -21,6 +21,7 @@ import (
 	"context"
 	"encoding/base64"
 	"fmt"
+	"github.com/apache/dubbo-go/common/constant"
```
Comment (zouyx) [inline cluster/router/condition/factory_test.go:24]: Split package

File: cluster/cluster_impl/failback_cluster_invoker.go (hunk 1 above)
Comment (georgehao) [reply cluster/cluster_impl/failback_cluster_invoker.go]: done

File: cluster/router/condition/app_router_test.go (hunk 2 above)
Comment (georgehao) [reply cluster/router/condition/app_router_test.go]: done

File: cluster/router/condition/factory_test.go (hunk 3 above)
Comment (georgehao) [reply cluster/router/condition/factory_test.go]: done

**Pull request merged**